src/content/development/contribution-guide/_index.en.md

```diff
@@ -62,7 +62,35 @@ To be expanded
 
 #### Feature Design
 
-To be expanded
+Project members are expected to work on feature design for any [epic](#epics) feature assigned to them.
+During this sprint, project members will update their respective epics with any work identified during the design phase.
```

Comment:
What sprint? It mentions "this sprint" w/o previously defining it. Also should we confine feature design  to a specific sprint or time frame? I'd think feature design could be done anytime.